dds/idl/be_global.cpp

```diff
@@ -434,6 +434,20 @@ BE_GlobalData::parse_args(long& i, char** av)
           av[i + 1]));
         idl_global->parse_args_exit(1);
       }
+    } else if (!std::strcmp(av[i], "--default-data-representation")) {
+      default_data_representation_.set_all(false);
```

Comment:
I missed this part. So yeah this would work, but you could only choose one representation, where as the annotation allows you to chose multiple representations.

Reply:
Yes, it only allows one default representation. I can change to let it allow more than one representation, but I don't know if providing that flexibility is really necessary. I'll leave this out as we discussed.

Reply:
It would be necessary to be able to specify multiple representations if we were really doing it, because that's what the annotation does, but yes leave it out for now.